crates/re_viewport/src/add_space_view_or_container_modal.rs

```diff
@@ -70,7 +71,32 @@ fn modal_ui(
     ];
 
     for (title, subtitle, kind) in container_data {
-        if row_ui(ui, icon_for_container_kind(&kind), title, subtitle).clicked() {
+        let target_container_kind = target_container
```

Comment:
maybe `parent_container_kind` is more descriptive?

Reply:
We have "target_container" all over the place in that file, so I'd keep it as is for consistency.